src/svg/components/Alerts.ts

```diff
@@ -76,7 +76,7 @@ class Alerts extends Component {
     }
 
     public clear(){
-        console.warn('TODO: Not yet implemented');
+        this.svg.selectAll('.alert').remove();
```

Comment:
We shouldn't remove the entire g.alert container, since it is a method for clearing and not for removing